features/devicepixelratio.yml

```
@@ -0,0 +1,6 @@
+name: devicepixelratio
```

```suggestion
name: devicePixelRatio
```
I think? we should camelCase it like the property?